### ISSUE TITLE
oneCRL2RevocationsTxt fails with "multiple-value config.GetRecordURL() in single-value context"

### ISSUE THREAD

**Issue by cr** [mozilla/OneCRL-Tools#43]
Current master is broken:

```bash
$ go get github.com/mozmark/OneCRL-Tools/oneCRL2RevocationsTxt
# github.com/mozmark/OneCRL-Tools/oneCRL2RevocationsTxt
/tmp/go/src/github.com/mozmark/OneCRL-Tools/oneCRL2RevocationsTxt/main.go:51: multiple-value config.GetRecordURL() in single-value context
```

**Comment by cr** [comment]
This is essentially capturing the bug that was causing #40.

**Comment by mozmark** [comment]
This should work now. @cr please let me know (and move the release tag as appropriate).

**Comment by jcjones** [comment]
Closed by adc15940802aae7a31dd4513ddb0ff97735038cc
